R/WINDOWS.md

```diff
@@ -20,19 +20,19 @@ license: |
 
 To build SparkR on Windows, the following steps are required
 
-1. Install R (>= 3.1) and [Rtools](http://cran.r-project.org/bin/windows/Rtools/). Make sure to
+1. Install R (>= 3.1) and [Rtools](https://cloud.r-project.org/bin/windows/Rtools/). Make sure to
```

Comment:
In this PR, this seems to be the only change to a better mirror site. +1.

Reply:
Yeah I could have included this with the other change too. This one is not a 'functional' change so I put it here.